specs/references/upload-fragment.md

```
@@ -0,0 +1,20 @@
+# Upload Fragment References
+
+External sources consulted for this spec:
+
+- Amazon S3 presigned URLs:
+  https://docs.aws.amazon.com/AmazonS3/latest/userguide/using-presigned-url.html
```

markdownlint-cli2 (0.20.0)

[warning] Bare URL used

(MD034, no-bare-urls)